DATA/UNIVERSE/SYSTEMS/Bw03/Bw03.ini

```diff
@@ -3025,3 +3025,216 @@ spacedust = asteroiddust
 spacedust_maxparticles = 50
 sort = 20
 pop_type = lootable_field
+
+;; Mineable Gas Pockets
+
+[Object]
+nickname = mineable_small_hydrcarbon_Bw03_01
+ids_name = 459177
+;res str
+; Hydrcarbon Gas Pocket
+ids_info = 460651
+;res html
+; \m\bHydrocarbon Gas Pocket\l\B
+; 
+; A concentrated and dense gaseous mass of hydrocarbon, can only be harvested with the appropriate equipment.\n
+pos = -6879.0281, -165.522, -15838.1621
+archetype = mineable_gas_pocket
+loadout = mineable_gas_small_hydrocarbons
+phantom_physics = true
+
+[Object]
+nickname = mineable_small_hydrcarbon_Bw03_02
+ids_name = 459177
+;res str
+; Hydrcarbon Gas Pocket
+ids_info = 460651
+;res html
+; \m\bHydrocarbon Gas Pocket\l\B
+; 
+; A concentrated and dense gaseous mass of hydrocarbon, can only be harvested with the appropriate equipment.\n

```

Comment:
The last \n on these texts is useless. This \n is only required in special cases where FL usually adds auto generated stats, like for guns.